src/util/dataJoin.js

```diff
@@ -46,7 +46,7 @@
             // enter
             // when container is a transition, entering elements fade in (from transparent to opaque)
             var enterSelection = updateSelection.enter()
-                .append(element) // <<<--- this is the secret sauce of this whole file
+                .insert(element) // <<<--- this is the secret sauce of this whole file (#528)
```

Comment:
Is it sufficient referencing 528 here, or would it be better to add a comment with explanation?